src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -48,11 +49,12 @@
 import org.elasticsearch.client.Client;
 import org.elasticsearch.common.logging.ESLogger;
 import org.elasticsearch.common.logging.Loggers;
+import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.index.IndexNotFoundException;
 import org.elasticsearch.search.SearchHit;
 import org.elasticsearch.transport.RemoteTransportException;
 
-public class KibanaSeed {
+public class KibanaSeed implements ConfigurationSettings {
```

Comment:
@jcantrill that seems to be the technique that other classes use in order to have access to the constants defined there such as OPENSHIFT_CONFIG_USE_COMMON_DATA_MODEL